Pico8Emulator/unit/graphics/GraphicsUnit.cs

```diff
@@ -0,0 +1,409 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
```

Comment:
These will have to be removed before release since it is a project requirement for it to be framework independent. I know we have performance issues right now, but fixing them while keeping independence needs to be our priority.

Reply:
I think a simple enough solution would be to create an abstract class that implements Flip. That way we can have one implementation for each framework / engine, having better performance while still maintaining independence. Then the user can specify which implementation to use based on where they are working on.

If someone wants to use the emulator with a framework that is not either Monogame, Unity or Godot (which I think are the main ones we should support right now), they can easily implement the Flip function and color conversions.